api/envoy/extensions/filters/http/composite/v3/composite.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.composite.v3;
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.composite.v3";
+option java_outer_classname = "CompositeProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Composite]
+// Composite Filter :ref:`configuration overview <config_http_filters_cors>`.
+// [#extension: envoy.filters.http.composite]
+
+// Composite filter config.
+message Composite {
+  // List of permissible types that may be provided via the matching API.
```

Comment:
Can you elaborate on this? It would be good to have some examples of configurations and how this works in the docs.

Reply:
I think this should go actually: this was meant to mirror what the ECDS validation around instantiating arbitrary filters, but since the entire configuration is coming from the same place I don't think this is necessary